pages/profile.js

```diff
@@ -3,187 +3,164 @@ import Link from 'next/link'
-        {t.profileAndSecuritySettings}
+        {t.profileSettings}
       </h1>
-      <TabList
-        titles={[t.profileAndPreferences, t.securitySettings]}
-        onClick={(index) => setTabSelected(index)}
-        tabSelected={tabSelected}
-        containerStyle={'flex flex-wrap border-b border-gray-200'}
-        selectedTabStyle={
-          'bg-gray-100 active border-b-2 border-b-bright-blue-light'
-        }
-        unselectedTabStyle={
-          'text-gray-500 hover:text-gray-600 hover:bg-gray-50'
-        }
-        genericTabStyle={'inline-block text-lg py-3 px-4 text-center'}
-        locale={props.locale}
-      />
-      {tabSelected === 0 ? (
-        <>
-          <TabList
-            titles={[t.ei, t.cpp, t.oas]}
-            onClick={(index) => setProfileTabSelected(index)}
-            tabSelected={profileTabSelected}
-            containerStyle={'my-4 flex flex-wrap gap-4'}
-            selectedTabStyle={'bg-bright-blue-lighter'}
-            unselectedTabStyle={'bg-gray-lighter'}
-            genericTabStyle={
-              'text-center px-10 py-1 whitespace-nowrap rounded-xl hover:bg-bright-blue-lighter '
-            }
-            locale={props.locale}
-          ></TabList>
-          {profileTabSelected === 0 || !profileTabSelected ? (
-            <ProfileInfo
-              fields={[fakeFieldsEI, fakeFieldsEI2]}
-              locale={props.locale}
-            />
-          ) : profileTabSelected === 1 ? (
-            <ProfileInfo fields={[fakeFieldsCPP]} locale={props.locale} />
-          ) : profileTabSelected === 2 ? (
-            <ProfileInfo fields={[fakeFieldsOAS]} locale={props.locale} />
-          ) : null}
-        </>
-      ) : tabSelected === 1 ? (
-        <ProfileInfo fields={[fakeSecurityFields]} locale={props.locale} />
-      ) : null}
+      <hr className="border-1 border-red-400" />
+      <span className="text-lg">{t.updateProfile}</span>
+
+      {benefitInformations.map((benefitInfo) => {
+        return (
+          <>
+            <h2 className="text-3xl py-2 font-bold">{benefitInfo.benefit}</h2>
+            <div className="py-2 mb-4">
+              {benefitInfo.show ? (
+                <ProfileInfo
+                  fields={benefitInfo.fields}
+                  locale={props.locale}
+                />
+              ) : null}
+              <ViewMoreLessButton
+                expanded={benefitInfo.show}
+                icon={benefitInfo.show}
+                onClick={() => benefitInfo.setShow(!benefitInfo.show)}
+                caption={!benefitInfo.show ? 'View settings' : 'View less'}
+              />
+            </div>
+
+            <hr className="border-1 border-gray-500" />
+          </>
+        )
+      })}
+
+      <div className="mt-10">
+        <h2 className="text-3xl font-bold">Looking for security settings?</h2>
+        <ul className="list-disc ml-8 text-lg">
+          <Link href={'/security-settings'}>
```

Comment:
Don't forget to add the `passHref`
```suggestion
          <Link href={'/security-settings'} passHref>
```

Reply:
Nice catch